docs/static/breaking-changes.asciidoc

```diff
@@ -1,5 +1,14 @@
 [[breaking-changes]]
-== Breaking Changes
+
+== Breaking Changes Across PQ Versions
+
+If you are upgrading Logstash and use the persistent queue, we strongly recommend that you drain or delete the persistent queue before you upgrade.
+
+To drain the queue, enable the `queue.drain` setting, and then shutdown Logstash. Wait for it to shutdown completely. This may take a while if you have a large queue backlog.
```

Comment:
Is this enough? What happens if the output is slower than the inputs, could we get into a situation where Logstash never stops?

Reply:
When you issue the shutdown command the inputs are all stopped, and then we wait for the queue to drain.